src/TestUtils/src/Microsoft.Maui.IntegrationTests/Utilities/DotnetInternal.cs

```diff
@@ -82,8 +102,10 @@ public static bool New(string shortName, string outputDirectory, string framewor
 		public static bool Run(string command, string args, int timeoutinSeconds = DEFAULT_TIMEOUT)
 		{
 			var runOutput = RunForOutput(command, args, out int exitCode, timeoutinSeconds);
-			if (exitCode != 0)
-				TestContext.WriteLine(runOutput);
+			TestContext.WriteLine($"Process exit code: {exitCode}");
+			TestContext.WriteLine($"-------- Process output start --------");
+			TestContext.WriteLine(runOutput);
```

Comment:
This produces more details in the console log output of the tests, which greatly aid in debugging tests even when they pass.